.github/workflows/build.yml

```diff
@@ -15,6 +15,9 @@ jobs:
         with:
           node-version: 20
 
+      - name: Get package version
```

Comment:
is it not necessary to change the version in package.json ?

Reply:
in this job, we are just running the tests, so no need to change the version, we just need to put the version in an env variable to be able to tag the commit